### PR TITLE
Add Tailwind CSS

### PR DESCRIPTION
**Pull request by willemvb**
- Add Tailwind CSS 0.3.0 on the front
- Remove front dummy styling
- Cleanup some class names
- Cleanup some views (focus on article text)
- Remove deprecated browser warning

### CODE REVIEW

File: resources/assets/css/front/bootstrap.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    &:after,
+    &:before {
+        box-sizing: inherit;
+    }
+}
+
+html {
+    box-sizing: border-box;
+    font-size: 16px;
```
Comment (sebastiandedeyne) [inline resources/assets/css/front/bootstrap.css]: Hmm not sure if this one's a good idea for accessibility https://www.sitepoint.com/stop-maiming-bodies-the-perils-of-pixel-font-size/

**Comment by sebastiandedeyne** [comment]
Looks good! I'll add purgeCss to this branch too so we can merge it all in one go.

**Comment by sebastiandedeyne** [comment]
I added in Purgecss. The Blender ignore rules could be cleaner in the future, but I bumped against a restriction of the Purgecss webpack plugin (see https://github.com/FullHuman/purgecss-webpack-plugin/issues/3).

A few other thoughts I have regarding files we touched in this PR...

- Is `mix.autoload` necessary? Can we remove this by default?
- Can we remove the `extraJs` Blade sections? Webpack does pretty much all our conditional JS loading
- Is Browsernizr still necessary? I wouldn't mind dropping the `head` scripts all together
- I personally don't use the images & downloads slots in `main`, if necessary I create an `article` component. I don't mind having them there though if you prefer

File: resources/assets/css/front/bootstrap.css (hunk 1 above)
Comment (willemvb) [reply resources/assets/css/front/bootstrap.css]: You're right, removed it

**Comment by willemvb** [comment]
- `mix.autoload`: it makes jQuery available globally if I'm correct. Not sure if this is still needed on blender scripts or redactor?
- Drop `extraJs`: it is still used to inject some chart data in Blender
- Browsernizr: let's drop it
- images & downloads slots: let's remove them by default

**Comment by sebastiandedeyne** [comment]
> `mix.autoload`: it makes jQuery available globally if I'm correct. Not sure if this is still needed on blender scripts or redactor?

I'm not sure, but we should keep it like that to be safe. Imo this is a flaw in Blender though, and shouldn't need to be part of our build pipeline. Instead I registered jQuery on `window` in `back/app.js`.

> Drop extraJs: it is still used to inject some chart data in Blender

Since the app script is deferred, and inline scripts aren't, it doesn't matter where the scripts that register `window.chartData` are placed. Removed the `extraJs` section since it isn't necessary.

**Pull request merged**